### ISSUE TITLE
Some pie charts get extra height while not needed

### ISSUE THREAD

**Issue by jimmykane** [amcharts/amcharts4#1538]
Take a look here 

<img width="404" alt="Screenshot 2019-08-07 at 23 12 14" src="https://user-images.githubusercontent.com/1268723/62658362-e9b1ed00-b968-11e9-8e21-8a8be9c33c4b.png">

The chart is drawn and I think the top anchor point is shown + the overflow. 

One of the categories is hidden due to being drawn higher and ourside the defined height.

**Comment by martynasma** [comment]
Yo, which version are you using?

I think we have fixed some related issue in one of the most recent versions.

**Comment by jimmykane** [comment]
Hola `4.5.13`. 

I am not sure if that was before but for me it exposed when I found the `group_slicer` plugin

**Comment by jimmykane** [comment]
I think before with just normal 'categories' I was fine but not 100% sure

**Comment by martynasma** [comment]
Could you post your chart config and data just so we're testing with exact setup?

**Comment by jimmykane** [comment]
Chart 
```
protected createChart(): am4charts.PieChart {
    const chart = <am4charts.PieChart>super.createChart(am4charts.PieChart);

    chart.hiddenState.properties.opacity = 0;
    // chart.padding(0, 0, 0, 0)
    chart.radius = am4core.percent(80);
    chart.innerRadius = am4core.percent(55);

    const pieSeries = chart.series.push(new am4charts.PieSeries());
    pieSeries.dataFields.value = 'value';
    pieSeries.dataFields.category = 'type';
    // pieSeries.interpolationDuration = 500;
    // pieSeries.rangeChangeDuration = 500;
    // pieSeries.sequencedInterpolation = true;

    pieSeries.slices.template.propertyFields.isActive = 'pulled';
    pieSeries.slices.template.strokeWidth = 0.4;
    pieSeries.slices.template.strokeOpacity = 1;
    pieSeries.slices.template.stroke = am4core.color('#175e84');


    const label = pieSeries.createChild(am4core.Label);
    label.horizontalCenter = 'middle';
    label.verticalCenter = 'middle';
    // label.fontSize = 12;
    if (this.chartDataValueType === ChartDataValueTypes.Total) {
      label.text = `{values.value.sum.formatNumber('#')}`;
    }
    if (this.chartDataValueType === ChartDataValueTypes.Maximum) {
      label.text = `{values.value.high.formatNumber('#')}`;
    }
    if (this.chartDataValueType === ChartDataValueTypes.Minimum) {
      label.text = `{values.value.low.formatNumber('#')}`;
    }
    if (this.chartDataValueType === ChartDataValueTypes.Average) {
      label.text = `{values.value.average.formatNumber('#')}`;
    }
```

Data

```
[
  {
    "type": "Multisport",
    "value": 28.299999999999997,
    "id": 0
  },
  {
    "type": "Flexibility Training",
    "value": 809.5799999999999,
    "id": 1
  },
  {
    "type": "Training",
    "value": 5800.33,
    "id": 2
  },
  {
    "type": "Strength Training",
    "value": 9196.779999999999,
    "id": 3
  },
  {
    "type": "Running",
    "value": 12700.439999999999,
    "id": 4
  },
  {
    "type": "Hiking",
    "value": 25610.34,
    "id": 5
  },
  {
    "type": "Trail Running",
    "value": 156566.88799999995,
    "id": 6
  }
]
```

**Comment by jimmykane** [comment]
Updated the above commend to show the plugin. 

Without the plugin
<img width="534" alt="Screenshot 2019-08-07 at 23 52 51" src="https://user-images.githubusercontent.com/1268723/62660510-81fea080-b96e-11e9-8b23-32fcadbb6636.png">

**Comment by martynasma** [comment]
Fixed in [4.5.14](https://github.com/amcharts/amcharts4/releases/tag/4.5.14).

## [4.5.14] - 2019-08-11

### Added
- `mouseOptions` added to `Interactions`. For now it contains only wheel sensitivity setting, e.g. `mapChart.interactions.mouseOptions.sensitivity = 0.4`.
- `getPositionX()` and `getPositionY()` methods added to all axes. Returns relative position of a series data item on axis.

### Changed
- Series with `DateAxis` not as a base axis of a series (like gantt) will now animate similar to other charts.
- `SlicedChart` hidden labels arrangements adjusted.

### Fixed
- `ValueAxis` was incorrectly choosing min/max values for the data with very small values.
- `StepLineSeries` with `connect = false` and `autoGapCount > 1` was showing regular line series, not a step series.
- Chart with `DateAxis` which was not a base axis of a series and dates older than Unix Epoch (January 1st, 1970 at UTC) was incorrectly showing base of a series.
- `DateAxis` was not animating smoothly when adding more data.
- `ValueAxis` pre-zoom (on `ready` event) was not working properly.
- `fullWidth` cursor line was not reverting back if set to `true` and then back to `false`.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
